test/intl402/NumberFormat/prototype/toStringTag/initial-value.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.numberformat.prototype-@@tostringtag
+description: >
+  Initial value of Intl.NumberFormat.prototype[@@toStringTag] is "Object".
+---*/
+
+assert.sameValue(Intl.NumberFormat.prototype[Symbol.toStringTag], 'Object');
+assert.sameValue(
+  Object.prototype.toString.call(new Intl.NumberFormat()),
+  '[object Object]'
+);
```

Comment:
Add tests that reconfigure the toStringTag and assert that's reflected with both a regular lookup and in a toString call similar to the above two tests